src/meta/processors/jobMan/TaskDescription.h

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_TASKDESCRIPTION_H_
+#define META_TASKDESCRIPTION_H_
+
+#include <ctime>
+#include <vector>
+#include <string>
+#include <folly/Range.h>
+#include <gtest/gtest_prod.h>
+#include "meta/processors/jobMan/JobStatus.h"
+
+/*
+ * =====================================================================================
+ * | Job Id(TaskId) | Command(Dest) | Status   | Start Time        | Stop Time         |
+ * =====================================================================================
+ * | 27             | flush nba     | finished | 12/09/19 11:09:40 | 12/09/19 11:09:40 |
+ * -------------------------------------------------------------------------------------
+ * | 27-0           | 192.168.8.5   | finished | 12/09/19 11:09:40 | 12/09/19 11:09:40 |
+ * -------------------------------------------------------------------------------------
+ * */
+
+namespace nebula {
+namespace meta {
+
+class TaskDescription {
+    FRIEND_TEST(TaskDescriptionTest, ctor);
+    FRIEND_TEST(TaskDescriptionTest, parseKey);
+    FRIEND_TEST(TaskDescriptionTest, parseVal);
+    FRIEND_TEST(TaskDescriptionTest, dump);
+    FRIEND_TEST(TaskDescriptionTest, ctor2);
+    FRIEND_TEST(JobManagerTest, showJob);
+
+public:
+    TaskDescription(int32_t iJob, int32_t iTask, const std::string& dest);
+    TaskDescription(const folly::StringPiece& key, const folly::StringPiece& val);
+
+    std::string taskKey();
+    static std::tuple<int32_t, int32_t> parseKey(const folly::StringPiece& rawKey);
+
+    std::string taskVal();
```

Comment:
Add comments for all public methods. Especially the Key/Value format.

Reply:
added